package.json

```diff
@@ -161,6 +161,7 @@
     "tap": "^14.4.1",
     "tap-mocha-reporter": "^5.0.0",
     "then-sleep": "^1.0.1",
+    "ts-standard": "^9.0.0",
```

Comment:
I thought that `tsd` could be removed with this: was I wrong?

Reply:
indeed, ts-standard only does the linting.